microhapdb/marker.py

```diff
@@ -430,3 +430,68 @@ def local_to_global(self, coord):
         if coord >= end - start:
             return None
         return coord + start
+
+
+class Locus:
+    def __init__(self, markers=None):
+        self.markers = list() if markers is None else markers
```

Comment:
Most of the code enabling this improved handling is part of this new `Locus` class. It replicates some of the features originally implemented for the `Marker` class. I considered removing those configuration features and only allowing configuration of e.g. delta, min length, etc., for Locus objects. But I think we need to maintain these features on a per-marker for other output modes.